_src/css/_secdarker.scss

```diff
@@ -0,0 +1,21 @@
+.section--darker {
+  background-color: #1E1E1E;
+
+  &__title {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__section {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-top: 50px;  
+  }  
+}
+
+#links {
```

Comment:
@felipprodrigues this looks good but to avoid inheritance, you could easily add a second class to `.box` like `.box--link` instead. This way we don't need two elements to be styled 🙃